gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-bin.zip
```

Comment:
I think you mean `4.10.2` here.  If updating gradle you will need to update the `build.upload` like this as well:

https://github.com/threadly/threadly/commit/2df9ce57f5390939b456b293be9fe3ff93f85c1a

Reply:
ok, I will get that done